lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -199,6 +199,51 @@ static void tryCopyName(Operation *dst, Operation *src) {
       dst->setAttr("sv.namehint", attr);
 }
 
+namespace {
+
+// A helper strutc to hold information about output file descriptor.
+struct FileDescriptorInfo {
```

Comment:
Nit: LLVM coding guidelines would say this should be a `class` since it has private members.